Anlab.Mvc/Client/components/AdditionalEmails.tsx

```diff
@@ -1,15 +1,20 @@
 ﻿import * as React from 'react';
 import Input from 'react-toolbox/lib/input';
 import { Button, IconButton } from 'react-toolbox/lib/button';
+import Chip from 'react-toolbox/lib/chip';
 
 interface IAdditionalEmailsProps {
     addedEmails: Array<string>;
     onEmailAdded: Function;
     onDeleteEmail: Function;
+    defaultEmail: string;
 }
 
 interface IAdditionalEmailsState {
     email: string;
+    toggle: boolean;
+    error: boolean;

```

Comment:
@laholstege since it's bool update the name to something like `HasError` or anything that has binary meaning, since otherwise error and errorText are a bit ambiguous